modal/app.py

```diff
@@ -1090,12 +1123,13 @@ def wrapper(wrapped_cls: Union[CLS_T, _PartialFunction]) -> CLS_T:
                 block_network=block_network,
                 restrict_modal_access=restrict_modal_access,
                 max_inputs=max_inputs,
+                http_config=http_config_proto,
                 scheduler_placement=scheduler_placement,
                 i6pn_enabled=i6pn_enabled,
                 cluster_size=cluster_size,
                 rdma=rdma,
                 include_source=include_source if include_source is not None else local_state.include_source_default,
-                experimental_options={k: str(v) for k, v in (experimental_options or {}).items()},
+                experimental_options={k: str(v) for k, v in (experimental_options_).items()},
```

Comment:
I'm a little unnerved by this change since the function is so long and it feels easy to miss that we have this "resolved" version of `experimental_options` if anyone else comes along and needs to inspect or mutate it.